go/tasks/plugins/hive/client/qubole_client.go

```diff
@@ -43,6 +43,11 @@ type QuboleCommandDetails struct {
 	URI    url.URL
 }
 
+type CommandMetadata struct {
+	TaskName string
+	Domain   string
```

Comment:
no project?

Reply:
How do i get project? This is useful for me :)

Reply:
I think in `execution_state.go` you can do `tCtx.TaskExecutionMetadata().GetTaskExecutionID().GetID().NodeExecutionId.ExecutionId.Project`